### DISCUSSION TITLE
How to merge upstream without all removal notices? (question only but maybe for howto)

### DISCUSSION THREAD

**Discussion by seansan** [OpenMage/magento-lts#2115] [Questions & Answers]
How to merge upstream without all removal notices? (question only but maybe for howto)

Command is `git pull upstream 1.9.4.x`

How can we prevent all these warning and continue? (our code base is M1 LTS )

![image](https://user-images.githubusercontent.com/652395/72525569-c028b600-3864-11ea-8f3e-ffdfc878a2b0.png)

**Comment by colinmollenhour** [comment]
I think this is just a side effect of forking the git repo directly and merging upstream commits. You can use `git add -A` to accept all of the unmerged paths at once, but just be careful that your working directory was clean before you ran the merge command.

**Comment by Flyingmana** [comment]
if you have resolved this once in a merge, it should only conflict again, if this files are affected by a change (which may be possible with the next patch release, if they again change the year in the header files)

And yes, this should get added to the HowTos
